source/renderer/app/i18n/locales/en-US.json

```diff
@@ -236,8 +236,8 @@
   "profile.settings.numberSelect.label": "Number format",
   "profile.settings.submitLabel": "Continue",
   "profile.settings.timeSelect.label": "Time format",
-  "profile.termsOfUse.checkboxLabel": "I agree with terms of use",
-  "profile.termsOfUse.checkboxLabelWithDisclaimer": "I agree with terms of use",
+  "profile.termsOfUse.checkboxLabel": "I agree with terms of service",
```

Comment:
@yakovkaravelov please update this key for Japanese translation too: https://github.com/input-output-hk/daedalus/blob/v2-integration/source/renderer/app/i18n/locales/ja-JP.json#L239